tests/test_client.py

```diff
@@ -1256,6 +1640,18 @@ def test_dos_get(client, user):
     assert rec_3["data_object"]["id"] == rec_1["did"]
 
 
+def test_get_dos_record_error(client, user):
+    data = get_doc(has_urls_metadata=True, has_metadata=True, has_baseid=True)
+
+    res = client.post("/index/", json=data, headers=user)
```

Comment:
this record you created is not used later in the test. you can remove it

Reply:
cool got it!